src/main/java/org/broadinstitute/hellbender/tools/spark/sv/BreakpointClusterer.java

```diff
@@ -61,15 +65,40 @@ private void buildTree( final Iterator<BreakpointEvidence> evidenceItr ) {
         treeItr = evidenceTree.iterator();
     }
 
-    private boolean hasEnoughOverlappers( final SVInterval interval ) {
+    @VisibleForTesting
+    protected boolean hasEnoughOverlappers( final SVInterval interval ) {
         final Iterator<SVIntervalTree.Entry<List<BreakpointEvidence>>> itr = evidenceTree.overlappers(interval);
-        int count = 0;
+        final SVIntervalTree<List<BreakpointEvidence>> targetIntervalTree = new SVIntervalTree<>();
+        int evidenceIntervalOverlapCount = 0;
         while ( itr.hasNext() ) {
-            count += itr.next().getValue().size();
-            if ( count >= minEvidenceCount ) {
+            final List<BreakpointEvidence> evidenceForInterval = itr.next().getValue();
+            evidenceIntervalOverlapCount += evidenceForInterval.size();
+            if ( evidenceIntervalOverlapCount >= minEvidenceCount ) {
                 return true;
             }
+
+            for (final BreakpointEvidence evidence : evidenceForInterval) {
+                if (evidence.hasDistalTargets()) {
+                    for (final SVInterval target : evidence.getDistalTargets(readMetadata)) {
```

Comment:
Not a comment against this commit, but I find these lines and lines 57-63 could be simplified if we have a `putIfAbsent()` method in `SVIntervalTree`